test/cookbooks/nextcloud-test/metadata.rb

```diff
@@ -0,0 +1,10 @@
+name              'nextcloud-test'
+maintainer        'Oregon State University'
+maintainer_email  'chef@osuosl.org'
+license           'All Rights Reserved'
+description       'proj-seagl nextcloud testing cookbook'
+version           '0.1.0'
+
+depends 'osl-nextcloud'
+depends 'osl-mysql'
+depends 'osl-apache'
```

Comment:
This should be all you need.
```suggestion
depends 'osl-mysql'
```